sdk/keyvault/azure-security-keyvault-keys/src/recover_deleted_key_operation.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/keyvault/keys/recover_deleted_key_operation.hpp"
+#include "azure/keyvault/keys/details/key_constants.hpp"
+#include "azure/keyvault/keys/details/key_serializers.hpp"
+
+using namespace Azure::Security::KeyVault::Keys;
+
+namespace {
+
+// For delete key, the LRO ends when we can retreive the Key from the deleted keys list from the
+// server.
+inline Azure::Core::OperationStatus CheckCompleted(Azure::Core::Http::RawResponse const& response)
+{
+  auto const code = response.GetStatusCode();
+  switch (code)
+  {
+    case Azure::Core::Http::HttpStatusCode::Ok:
+    case Azure::Core::Http::HttpStatusCode::Forbidden: // Access denied but proof the key was
+                                                       // deleted.
```

Comment:
Nit, this white space is confusing.  I'd put the comment above both of the cases